js/aioseop-helper.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * Use for JavaScript compatability by using anonymous functions; which
+ * are loaded into the footer.
+ *
+ * @summary   AIOSEOP Footer JS
+ *
+ * @since     2.4.2
+ */
+// phpcs:disable PEAR.Functions.FunctionCallSignature.ContentAfterOpenBracket
+// phpcs:disable PEAR.Functions.FunctionCallSignature.MultipleArguments
+// phpcs:disable PEAR.Functions.FunctionCallSignature.CloseBracketLine
+(function($) {
+	/**
+	 * jQuery UI Tooltips
+	 *
+	 * Initiates the jQuery UI Tooltips on the admin pages with the class and
+	 * title attributes set properly.
+	 *
+	 * @since 2.4.1.1
+	 *
+	 * @link http://api.jqueryui.com/tooltip/
+	 */
+	function aioseopTooltips() {
```

Comment:
@wpsmort I have faced an (as yet unresolved) conflict with themes that use twitter bootstrap as that interferes with jquery tooltips (https://stackoverflow.com/questions/13731400/jqueryui-tooltips-are-competing-with-twitter-bootstrap). It would be good to test with that as some of the most popular themes use that. I don't remember the theme name. cc @EkoJR

Reply:
This is something that I am aware of, and tried to keep in mind while coding. It probably would be worth testing.